rescript

```diff
@@ -41,31 +34,17 @@ if (fs.existsSync(bsConfigFile)) {
   }
 }
 
-// All clients of type MiniWebSocket
-/**
- * @type {any[]}
- */
-var wsClients = [];
-var isWatchMode = false;
-var verbose = false;
-/**
- * @type {string | undefined}
- */
-var postBuild = undefined;
```

Comment:
The `postBuild` wasn't used, so I removed it. The `isWatchMode` also wasn't needed anymore and was removed. The other code is moved without changes.